### PR TITLE
Update configuration page

### PR DESCRIPTION
**Pull request by gwdawson**
this pr updates the configuration to follow the best practices.

Partial [#72761](https://github.com/grafana/grafana/issues/72761)
Fixes [#244](https://github.com/grafana/github-datasource/issues/244)

**Before & After:**
Screenshot 2023-10-04 at 13 16 24

https://github.com/grafana/github-datasource/assets/34524710/19ff4584-e172-4840-a2db-9cb97e6f2529

### CODE REVIEW

**Comment by zoltanbedi** [comment]
Can we have the collapsed info on the top not to be collapsed by default? I also think that the old info box was more attention grabber. The radio button for the license doesn't really make sense for me.

Can we also have @catherineymgui or someone from UX to also review these changes?

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-16
\ No newline at end of file
+lts/*
```
Comment (zoltanbedi) [inline .nvmrc:1]:
```suggestion
18
```

**Review by gabor — APPROVED** [review]
LGTM 👍 

@zoltanbedi WDYT?

**Comment by zoltanbedi** [comment]
The collapsible info section at the top is still collapsed by default also in the doc the additional settings are collapsed. I'm still not fan of the radio button but I can live with that. we can try it out and see.

**Comment by gwdawson** [comment]
> The collapsible info section at the top is still collapsed by default also in the doc the additional settings are collapsed. I'm still not fan of the radio button but I can live with that. we can try it out and see.

@zoltanbedi edited to made additional settings collapsible, also as requested above i have set the "Access Token Permissions" to be open by default.

**Comment by catherineymgui** [comment]
Hey sorry I was on PTO! Radio button list might make more sense if you wanted to iterate on this.

https://developers.grafana.com/ui/latest/index.html?path=/story/forms-radiobuttonlist--controlled-component

**Pull request merged**